docs/Containers/Zigbee2MQTT.md

```diff
@@ -1,262 +1,463 @@
 # Zigbee2MQTT
 
+## Quick links
+
+* New users: [start here](#basicProcess)
+* Existing users: [Service definition change](#update202204) (circa April 2022)
+
+## References
+
 * [Web Guide](https://www.zigbee2mqtt.io)
-* [Flashing the CC2531](https://www.zigbee2mqtt.io/information/flashing_the_cc2531.html)
-* [Figuring-out your device identifier](https://www.zigbee2mqtt.io/getting_started/running_zigbee2mqtt.html)
+* [Supported adapters](https://www.zigbee2mqtt.io/guide/adapters/#recommended)
+* [GitHub](https://github.com/Koenkk/zigbee2mqtt)
+* [DockerHub](https://hub.docker.com/r/koenkk/zigbee2mqtt/tags)
 
-## Service definition change - April 2021
+## Definitions
 
-The IOTstack service definition for Zigbee2MQTT is at the following path:
+* *"compose file"* means the file at the path:
 
-```
-~/IOTstack/.templates/zigbee2mqtt/service.yml
-```
+	```
+	~/IOTstack/docker-compose.yml
+	```
+
+## <a name="basicProcess"></a>Basic process for new users
+
+1. Run the IOTstack menu and choose both "Mosquitto" and "Zigbee2MQTT". That adds the service definitions for both of those containers to your *compose file*.
 
-As of April 2021, the service definition changed:
+2. [Prepare your Zigbee adapter](#prepareAdapter) by flashing its firmware.
+3. Follow the steps in [Identify your Zigbee adapter](#identifyAdapter) to work out how your adapter "mounts" on your Raspberry Pi, and edit your *compose file* to include that information.
+4. The default environment variables assume:
 
-1. The Zigbee2MQTT container no longer runs in host mode.
-2. Adds timezone support.
-3. Builds the container from a Dockerfile providing appropriate defaults for IOTstack.
-4. Re-adds a port mapping for port 8080 (the Zigbee2MQTT web UI).
+	- You are running Mosquitto and Zigbee2MQTT as IOTstack containers on the same computer; and
+	- You want the Zigbee2MQTT web front end to be available on port 8080.
 
-If you were running the Zigbee2MQTT service before this change, you may wish to compare and contrast your active service definition (in `docker-compose.yml`) with the revised template.
+	This is a good basis for getting started. If it sounds like it will meet your needs, you will not need to make any changes. Otherwise, review the [environment variables](#envVars) and make appropriate changes to the service definition in your *compose file*.
+
+5. <a name="upStack"></a>Bring up your stack:
+
+	```bash
```

Comment:
In regards to #547, could you change this and later code-blocks from `bash` to `console`?

Reply:
Done.